lib/transform/transformers.js

```diff
@@ -31,10 +31,7 @@ export function assets (asset) {
     asset.fields.file,
      (newFile, file, locale) => {
        newFile[locale] = omit(file, 'url', 'details')
```

Comment:
Instead of blacklisting properties I would explicitly say the ones we need:

```
pick(file, `contentType`, `fileName`)
```

Because otherwise we forgot to also omit `upload`